ui/components/component-library/base-icon/base-icon.scss

```diff
@@ -0,0 +1,34 @@
+.base-icon {
+  --icon-size: var(--size, 16px);
+
+  &--size-xxs {
+    --size: 10px;
+  }
+
+  &--size-xs {
+    --size: 12px;
+  }
+
+  &--size-sm {
+    --size: 16px;
+  }
+
+  &--size-md {
+    --size: 20px;
+  }
+
+  &--size-lg {
+    --size: 24px;
+  }
+
+  &--size-xl {
+    --size: 32px;
+  }
+
+  font-size: var(--icon-size);
+  width: 1em;
+  height: 1em;
+  display: inline-block;
+  fill: currentColor;
+  color: inherit;
```

Comment:
Can remove this now that Box supports inherit 
```suggestion
```

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/15609/commits/8b173b3ba673d1a4ef2b82e93dc4f5eb8fd4a93c